### ISSUE TITLE
Remove incorrect photo

### ISSUE THREAD

**Issue by openui54u** [openchargemap/ocm-system#218]
Just added a photo to wrong location. Csn not find easy removal of summitted photo. Nor can suggestions be made to remove. Tag as incorrect or such.

**Comment by webprofusion-chrisc** [comment]
You can manage your media uploads here: https://openchargemap.org/site/profile/media but yes we do need more functionality in the app/api for that.

**Comment by webprofusion-chrisc** [comment]
@soilsong yep sure, do you mean in the web app (which is what most people use) or the site page like https://openchargemap.org/poi/details/474380 - you can already upload photos but removing them is under the my profile menu on the website. 

Ideally removing a photo would also delete it from our cloud storage, but just database entry is fine.

Only an admin or the original user should be able to delete a photo.
